File: 정렬/전승훈/boj_11650_좌표 정렬하기.swift
let n = Int(readLine()!)!

var arr: [(Int, Int)] = []

for _ in 0..<n {
    let input = readLine()!.split(separator: " ").map{ Int($0)! }
    let x = input[0]
    let y = input[1]
    
    arr.append((x, y))
}

let sortedArr = arr.sorted { $0.0 == $1.0 ? $0.1 < $1.1 : $0.0 < $1.0 }

for i in sortedArr {
    print("\(i.0) \(i.1)")
}

File: 정렬/박성수/boj_11650_좌표정렬하기.py
# https://www.acmicpc.net/problem/11650
# 정렬순을 여러가지 기준을 줄 때, tuple 형식으로 값을 전달할 수 있음

N = int(input())

arr = []
for _ in range(N):
    arr.append(list(map(int, input().split())))

arr.sort()
# arr.sort(key=lambda x: (x[0], x[1]))

for i in arr:
    print(i[0], i[1])
